.github/actions/integration-test/action.yaml

```diff
@@ -0,0 +1,45 @@
+name: integration-test
+description: 'action that prepares the integration test setup and runs the provided integration test script'
+inputs:
+  integration-test-script:
+    description: 'file name of integration test script'
+    required: true
+    default: 'integration-test.sh'
```

Comment:
Does one need to set a default here? Otherwise I'd just leave the default out

Looks really nice, overall +1

Reply:
removed that line :-) Thanks!